mobile/src/base/java/be/mygod/vpnhotspot/util/FabricUtil.kt

```diff
@@ -0,0 +1,16 @@
+package be.mygod.vpnhotspot.util
+
+import android.content.Context
+import be.mygod.vpnhotspot.BuildConfig
+import com.crashlytics.android.Crashlytics
+import io.fabric.sdk.android.Fabric
+import timber.log.Timber
+
+object FabricUtil {
+    fun init(context: Context) {
+        if (!BuildConfig.DEBUG) {
+            Fabric.with(context, Crashlytics())
+            Timber.plant(CrashlyticsTree())
+        }
```

Comment:
Wouldn't that be redundant? As I can see, according to the Crashlytics docs, "In addition to writing to the next crash report, it will also write to the LogCat using android.util.Log.println(priority, tag, msg)"

Reply:
It's done 👍